web/src/main/resources/application.properties.EXAMPLE

```diff
@@ -14,6 +14,8 @@ vep.max_page_size=200
 # This is only the VEP part of the Ensembl REST API.
 # e.g. when running on port 6060:
 # gn_vep.region.url=http://localhost:6060/vep/human/region/VARIANT
+# gn_vep.cache.version=
+# gn_vep.server.version=
```

Comment:
@averyniceday @sheridancbio: it would be nice to add some comments on when and why to set these properties.

(we are currently going through the latest PRs to understand what has changed)